src/main/java/aztech/modern_industrialization/machines/init/SingleBlockSpecialMachines.java

```diff
@@ -23,6 +23,8 @@
  */
 package aztech.modern_industrialization.machines.init;
 
+import static aztech.modern_industrialization.machines.init.MachineRegistrationHelper.*;
```

Comment:
Please remove this, it's quite a large diff for no reason. :P

Reply:
Though I must say it is quite nicer without the MachineRegistrationHelper, so this is fine actually.